charts/kof-mothership/templates/grafana/dashboards/dashboard.yaml

```diff
@@ -6,6 +6,8 @@
 {{- $dashboardName := (trimSuffix (ext $dashboardBase) $dashboardBase) -}}
 {{- $dashboardData := fromYaml (tpl ($.Files.Get $dashboardFile) $ctx) -}}
 {{- $dashboardResult := toRawJson $dashboardData }}
+{{- $folderName := (split "-" $dashboardName)._0 }}
```

Comment:
That works now, but IMHO would be more explicit to move files to the corresponding folders.
@denis-ryzhkov WDYT?